Apps/Admin/Client/Pages/AgentAccessPage.razor.cs

```diff
@@ -35,6 +35,8 @@ namespace HealthGateway.Admin.Client.Pages;
 /// </summary>
 public partial class AgentAccessPage : FluxorComponent
 {
+    private const int SearchResultLimit = 25;
```

Comment:
Should this and AgentAccessService.GetAgentsAsync use the same value?

Reply:
I considered it, but didn't think it was worth adding a new class to store this one value.